docs/how-to/upgradePingAccess.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Upgrading PingAccess
+---
+
+# Upgrading PingAccess
+
+In a DevOps environment, upgrades can be simplified through automation, orchestration, and separation of concerns.
+
+!!! warning "Notice"
+    * Upgrading from PingAccess versions prior to 6.3.6 will not work using this method.
+
+## Caveats
+
+1.  **This Document Assumes Kubernetes and Helm**
+
+    The terms in this document will focus on deployments in a Kubernetes Environment using the ping-devops Helm chart. However, the concepts should apply to any containerized PingAccess Deployment.
+
+1.  **This Document will Become Outdated**
+
+    The examples referenced in this document point to a specific tag. This tag may not exist anymore at the time of reading. To correct the issue, update the tag on your file to N -1 from the current PF version. 
+
+1.  **Upgrades from Traditional Deployment**
+
+    It may be desirable to upgrade PingAccess along with migrating from a traditional environment. This is not recommended. Instead you should upgrade your current environment to the desired version of PingAccess and then [create a profile](./buildPingAccessProfile.md) that can be used in a containerized deployment.
```

Comment:
... environment.  Upgrading in this manner is not recommended.  Instead, you should .... (add comma, too)